build.sbt

```diff
@@ -121,7 +121,7 @@ lazy val cassandra =
 lazy val couchbase =
   alpakkaProject("couchbase", "couchbase", Dependencies.Couchbase, whitesourceGroup := Whitesource.Group.Supported)
 
-lazy val csv = alpakkaProject("csv", "csv", whitesourceGroup := Whitesource.Group.Supported)
+lazy val csv = alpakkaProject("csv", "csv", whitesourceGroup := Whitesource.Group.Supported, fatalWarnings := false)
```

Comment:
Do we want `fatalWarnings=false` in a supported module?

Reply:
Boolean logic it too much for me.